tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,178 @@
+  * - XR_ENV_BLEND_MODE_ADDITIVE
+    - The rendered image is added to the real world. This mode generally applies to AR only devices
+      such as the Hololens where the optics are not able to obscure the real world.
+      This will turn on passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ALPHA_BLEND
+    - The rendered image is alpha blended with the real world.
+      This is supported on devices that can fully obscure the real world
+      and generally the preferred way to do AR.
+      This will turn on passthrough if passthrough is used.
+  
+You can set the environment blend mode for your application through the ``environment_blend_mode``
+property of the :ref:`XRInterface <class_xrinterface>` instance.
+
+You can query the supported blend modes on the hardware using the
+``get_supported_environment_blend_modes`` property on the same instance.
+
+Configuring your background
+---------------------------
+
+When setting the blend mode to ``XR_ENV_BLEND_MODE_ALPHA_BLEND`` you must set
+the ``transparent_bg`` property on :ref:`Viewport <class_viewport>` to true.
+It is advisable to do the same with ``XR_ENV_BLEND_MODE_ADDITIVE`` however here you
+can alternatively set your background color to black.
+
+Either solution will result in the background rendering not contributing to lighting.
+It is thus also recommended you adjust your environment settings accordingly and ensure
+there is adequate ambient light set to illuminate your scene.
+
+.. note::
+
+  Some AR SDKs do provide ambient lighting information or even provide a full radiance
+  map to allow for real world reflections in your virtual objects.
+  The core Godot XR functionality doesn't currently have support for this however this
+  functionality can be exposed through plugins.
+
+OpenXR specific
+---------------
+
+In OpenXR you can configure the default blend mode you want to use.
+Godot will select this blend mode at startup if available.
+If not available Godot will default to the first supported blend mode provided
+by the XR runtime.
+
+.. image:: img/openxr_default_blend_mode.webp
+
+For passthrough devices OpenXR requires additional settings to be configured.
+These settings are platform dependent and provided through the OpenXR vendors plugin.
+
+For example, these are the settings required on Meta Quest:
+
+.. image:: img/openxr_export_passthrough.webp
+
+The ``Passthrough`` setting defines whether passthrough is supported or even required.
+
+The ``Boundary Mode`` allows you to define whether the guardian is needed,
+disabling this fully requires passthrough to be enabled at all times.
+
+Putting it together
+-------------------
+
+Putting the above together we can use the following code as a base:
+
+.. code-block:: gdscript
+
+  @onready var viewport : Viewport = get_viewport()
+  @onready var environment : Environment = $WorldEnvironment.environment
+
+  func switch_to_ar() -> bool:
+    var xr_interface: XRInterface = XRServer.primary_interface
```

Comment:
```suggestion
      var xr_interface: XRInterface = XRServer.primary_interface
```
Four spaces for indentation, throughout

Reply:
4 spaces didn't seem to work nicely for the ordered lists, but everything else now uses this indentation.

Reply:
Absolutely, meant specifically for GDScript